src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.main_header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo_img {
+  width: 40px;
+  height: 40px;
+}
```

Comment:
Avoid fixing the size of containers like `.logo_img` unless it's a requirement. If the image needs to be a specific size, consider using CSS properties like `max-width` or `max-height` to make the layout more flexible and to avoid overflow or accidental scroll bars.